src/query/service/src/interpreters/interpreter_network_policy_desc.rs

```diff
@@ -45,7 +45,7 @@ impl Interpreter for DescNetworkPolicyInterpreter {
     }
 
     fn is_ddl(&self) -> bool {
-        true
+        false
```

Comment:
Good and I do a fix in this pr: https://github.com/TCeason/datafuse/pull/new/fix_is_ddl